packages/@uppy/provider-views/src/ProviderView/ProviderView.jsx

```diff
@@ -99,32 +129,34 @@ export default class ProviderView extends View {
   }
 
   /**
-   * Based on folder ID, fetch a new folder and update it to state
+   * Select a folder based on its id: fetches the folder and then updates state with its contents
+   * TODO rename to something better like selectFolder or navigateToFolder (breaking change?)
```

Comment:
I don't think we need this todo? Name is not that bad.

Reply:
Unfortunately, almost everything in Uppy is state and almost all functions perform things on state. But yes since it's a public function I don't think we can change it now.